src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -1112,8 +1121,50 @@ public Task GetOngoingTaskInfo()
 
             return Task.CompletedTask;
         }
+        
+        [RavenAction("/databases/*/tasks/hub-pull-replication", "GET", AuthorizationStatus.ValidUser)]
+        public Task GetHubTasksInfo()
+        {
+            if (ResourceNameValidator.IsValidResourceName(Database.Name, ServerStore.Configuration.Core.DataDirectory.FullPath, out string errorMessage) == false)
+                throw new BadRequestException(errorMessage);
+
+            var key = GetLongQueryString("key");
```

Comment:
`taskId` instead of `key` ?

Reply:
we use key in every other ongoing task so it is by convention.